_posts/2018-11-16-grpc-dotnet-build.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: gRPC Meets .NET SDK And Visual Studio
+published: true
+permalink: blog/grpc-dotnet-build
+author: Kirill 'kkm' Katsnelson
+author-link: https://github.com/kkm000
+company: SmartAction
+company-link: https://www.smartaction.ai
+---
+As part of Microsoft's move towards its cross-platform .NET offering, they have
+greatly simplified the project file format, and allowed a tight integration of
+third-party code generators with .NET projects. We are listening, and now proud
+to introduce integrated compilation of Protocol Buffer and gRPC service
+`.proto` files in .NET C# projects starting with the version 1.17 of the
+Grpc.Tools NuGet package, now available from Nuget.org.
+
+<!--more-->
+
+## A Walkthrough
+
+Now you can include `.proto` files along with C# source files into `.csproj`
+project files. In this blog post, we'll walk through the simplest and probably
```

Comment:
Can we add a line stating why this is an improvement over the current situation, and the friction it removes? 

Having that will motivate the reader more to read into the details..

Reply:
Good point, thanks! I added a second paragraph from the top before the separator, please check if it explains that idea.